src/lib/utils.ts

```diff
@@ -52,3 +52,5 @@ export const isToken = (assetReference: AssetReference | string) =>
 
 export const tokenOrUndefined = (assetReference: AssetReference | string) =>
   isToken(assetReference) ? assetReference : undefined
+
+export const isDefined = <T>(optional: T | undefined): optional is T => Boolean(optional)
```

Comment:
Nice one! I literally also had to write this type guard inline a few hours ago, think we'll need it a lot 🤌

Reply:
`Boolean(0)` with `accountNumber = 0` here returns `false`, got bitten by this yesterday

we may need a mix of lodash's `isNull` and `isUndefined` or similar?

Reply:
dum language

Reply:
https://github.com/shapeshift/web/pull/3241/files#diff-da10bcc7d8dff796f50c9358a7268eb5abd2c50f460aecc4b2be1c691a966224R57

@0xApotheosis you might find this useful

Reply:
Right you are - truthy is the wrong comparison.

Used lodash in https://github.com/shapeshift/web/pull/3243/commits/6eeae7b3eb62475862935527d4a2c0d9c9bfd97a.